core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-announcements/dot-toolbar-announcements.component.html

```diff
@@ -1,67 +1,78 @@
+                            item.announcementDateAsISO8601 | date
+                        }}</span>
+                    </div>
+                </a>
+            </li>
+            }
+        </ul>
 
-<div class="announcements__container">
-    <div class="announcements__link-container">
-        <i class="pi pi-external-link"></i>
-        <a
-            class="announcements__link"
-            [href]="linkToDotCms()"
-            target="_blank"
-            data-testId="announcement_link_all"
-            rel="noopener"
-            >{{ 'announcements.show.all' | dm }}</a
-        >
-    </div>
-</div>
+        <div class="announcements__container">
+            <div class="announcements__link-container">
+                <i class="pi pi-external-link"></i>
+                <a
+                    class="announcements__link"
+                    [href]="linkToDotCms()"
+                    target="_blank"
+                    data-testId="announcement_link_all"
+                    rel="noopener"
+                    >{{ 'announcements.show.all' | dm }}</a
+                >
+            </div>
+        </div>
 
-<h5 class="announcements__title">{{ 'announcements.knowledge.center' | dm }}</h5>
-<div class="announcements__about">
-    @for (item of knowledgeCenterLinks(); track item.id;) {
-    <a
-        class="announcements__about-link"
-        [href]="item.url"
-        data-testId="announcement_link"
-        target="_blank"
-        rel="noopener"
-        >{{ item.label }}</a
-    >
-    }
-</div>
+        <h5 class="announcements__title">{{ 'announcements.knowledge.center' | dm }}</h5>
+        <div class="announcements__about">
+            @for (item of knowledgeCenterLinks(); track item.id;) {
+            <a
+                class="announcements__about-link"
+                [href]="item.url"
+                data-testId="announcement_link"
+                target="_blank"
+                rel="noopener"
+                >{{ item.label }}</a
+            >
+            }
+        </div>
 
-<h5 class="announcements__title">{{ 'announcements.knowledge.contact.us' | dm }}</h5>
-<div class="announcements__about">
-    @for (item of contactLinks(); track item.id;) {
-    <a
-        class="announcements__about-link"
-        [href]="item.url"
-        data-testId="announcement_link"
-        target="_blank"
-        rel="noopener"
-        >{{ item.label }}</a
-    >
-    }
-</div>
+        <h5 class="announcements__title">{{ 'announcements.knowledge.contact.us' | dm }}</h5>
+        <div class="announcements__about">
+            @for (item of contactLinks(); track item.id;) {
+            <a
+                class="announcements__about-link"
+                [href]="item.url"
+                data-testId="announcement_link"
+                target="_blank"
+                rel="noopener"
+                >{{ item.label }}</a
+            >
+            }
+        </div>
+    </div>
```

Comment:
It's a good idea to unify this by using an array of arrays. For example:

```ts
[
    { title: "announcements.knowledge.center", items: knowledgeCenterLinks() },
    { title: "announcements.knowledge.contact.us", items: contactLinks() }
];
```
It might be a good idea since the HTML is identical